src/main/java/Schmidt.java

```diff
@@ -0,0 +1,219 @@
+                "------------------------------------------------------------";
+        ArrayList<Task> tasks = new ArrayList<Task>();
+
+        System.out.println("Hello from\n" + LOGO);
+
+        System.out.println(HELLO_MESSAGE);
+
+        Scanner sc = new Scanner(System.in);
+
+        while (true) {
+            System.out.print("\t-> ");
+
+            String input = sc.nextLine();
+            String command = input.split(" ")[0];
+
+            // Switch statement to handle different commands
+            switch (command) {
+            case "bye":
+                printByeMessage();
+                return;
+            case "list":
+                printListHelper(tasks);
+                continue;
+            case "mark":
+                markTaskHelper(tasks, input, true);
+                continue;
+            case "unmark":
+                markTaskHelper(tasks, input, false);
+                continue;
+            case "todo":
+                addTodoHelper(tasks, input);
+                continue;
+            case "deadline":
+                addDeadlineHelper(tasks, input);
+                continue;
+            case "event":
+                addEventHelper(tasks, input);
+                continue;
+            default:
+                printValidCommands();
+                continue;
+            }
+        }
+    }
+
+    /**
+     * This is a function to print the valid commands
+     */
+    public static void printValidCommands() {
+        System.out.println("------------------------------------------------------------\n" +
+                " I'm sorry, but the valid commands are:\n" +
+                "\tbye\n" +
+                "\tlist\n" +
+                "\ttodo <description>\n" +
+                "\tdeadline <description> /by <date>\n" +
+                "\tevent <description> /from <date> /to <date>\n" +
+                "\tmark <task number>\n" +
+                "\tunmark <task number>\n" +
+                "------------------------------------------------------------");
+    }
+
+    /**
+     * This is a function to print the bye message
+     */
+    public static void printByeMessage() {
+        System.out.println("------------------------------------------------------------\n" +
+                " Bye. Hope to see you again soon!\n" +
+                "------------------------------------------------------------");
+    }
+
+    /**
+     * This is a helper method to print the list of tasks
+     * @param tasks The list of tasks
+     */
+    public static void printListHelper(ArrayList<Task> tasks) {
+        System.out.println("------------------------------------------------------------\n" +
+                " Here are the tasks in your list:");
+
+        for (int i = 0; i < tasks.size(); i++) {
+            System.out.println("\t" + (i + 1) + "." + tasks.get(i).toString());
+        }
+
+        System.out.println("------------------------------------------------------------");
```

Comment:
Is it better to add a method to print line to make the code look more clean?

Reply:
that's a great idea. i'll do that